src/main/java/com/jakub/todoSandbox/service/TodoService.java

```diff
@@ -0,0 +1,146 @@
+package com.jakub.todoSandbox.service;
+
+import com.jakub.todoSandbox.model.Step;
+import com.jakub.todoSandbox.model.Todo;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+public class TodoService {
+
+    // Temporary Map for todos - for testing purposes TODO implement postgress database for todos and steps instead of map
+    private Map<Long, Todo> todoList;
+
+    public TodoService(Map<Long, Todo> todoList) {
+        this.todoList = todoList;
+    }
+
+    public ModelAndView isListEmpty() {
+        if(todoList.isEmpty()) {
+            System.out.println("List is empty, returned to index.html");
+            return new ModelAndView("index.html");
+        } else {
+            System.out.println("List is not empty, forwarding to todo-list");
+            return new ModelAndView("/todo-list");
+        }
+    }
+
+    public void createTodo(Todo todo) {
+        todo.setId(todoList.size() + 1L);
+        todoList.put(todo.getId(),todo);
+        System.out.println(validateNameAndDesc(todo.getName(), todo.getDescription()));
```

Comment:
What this validation prevents us to todo? User can still create invalid TODO and they don't even know that it's invalid ;(

Reply:
That's completely different issue. What are you validating here? You create Todo, without checking whether it is correct or not and then you are checking whether it is correct and just log the result. What value does it provide?